src/templates/components/accordion/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Accordion as AccordionType } from '@/types/formatted/accordion'
+import { stringToId } from '@/lib/utils/stringToId'
+
+type AccordionProps = {
+  id: string
+  bordered: boolean
+  items: AccordionType[]
```

Comment:
can we call this FormattedAccordion instead? Matches the style used elsewhere (e.g. in `templates/common/button/index.tsx`)

Reply:
Good catch, fixed